### DISCUSSION TITLE
`denote-rename-file` in dired does not use the current file name

### DISCUSSION THREAD

**Discussion by brabalan** [protesilaos/denote#353] [General]
I have an inbox where some files land before integrating my denote folder. I try to give them a meaningful name when I save them, and I later rename them and tag them using `denote-rename-file`. I recently noticed that the current file name is not proposed by default for the file title, so I have to retype it all the time. Is there a way to call `denote-rename-file` such that the default title is not empty but the current file name?

**Comment by protesilaos** [comment]
> From: Alan Schmitt ***@***.***>
> Date: Fri, 17 May 2024 02:24:54 -0700
>
> I have an inbox where some files land before integrating my denote
> folder. I try to give them a meaningful name when I save them, and I
> later rename them and tag them using `denote-rename-file`. I recently
> noticed that the current file name is not proposed by default for the
> file title, so I have to retype it all the time. Is there a way to
> call `denote-rename-file` such that the default title is not empty but
> the current file name?

This is a regression from a previous commit I made. I have fixed it now.

I guess this is not really a "discussion" and should probably be in the
issue tracker. But it is fine.

-- 
Protesilaos Stavrou
https://protesilaos.com

**Comment by brabalan** [comment]
Thank you! I confirm it’s working great.

I’ll open an issue next time, sorry about that.

**Comment by protesilaos** [comment]
> From: Alan Schmitt ***@***.***>
> Date: Fri, 17 May 2024 03:42:37 -0700
>
> Thank you! I confirm it’s working great.

Very well!

> I’ll open an issue next time, sorry about that.

No worries!

-- 
Protesilaos Stavrou
https://protesilaos.com
